app/bundles/FormBundle/EventListener/PageSubscriber.php

```diff
@@ -70,8 +93,8 @@ public function onPageBuild(PageBuilderEvent $event)
         }
 
         if ($event->tokensRequested($this->formRegex)) {
-            $tokenHelper = new BuilderTokenHelper($this->factory, 'form');
-            $event->addTokensFromHelper($tokenHelper, $this->formRegex, 'name', 'id');
+            $tokenHelper = new BuilderTokenHelper($this->mauticFactory, 'form');
+            $event->addTokensFromHelper($tokenHelper, $this->formRegex, 'name', 'id', true);
```

Comment:
Why did this `true` get added? The 5th argument defaults to false

Reply:
I think this was some bad merge conflict resolution probably from this commit: https://github.com/mautic/mautic/commit/bc5021f8bc96a918533da8b9397a17a158889670

Will remove the `true` param.

Reply:
https://github.com/mautic/mautic/pull/8148/commits/5c792ae1c5bd4f9e04e6ee1caef50fda13849f27